config/config.go

```diff
@@ -413,6 +415,12 @@ type StmtSummary struct {
 	MaxSQLLength uint `toml:"max-sql-length" json:"max-sql-length"`
 }
 
+// Experimental controls features that are still experimental. Users should not rely on these features in a production environment.
+type Experimental struct {
+	// Whether allow to use the syntax like `auto_shard_bits(3)` on the primary key column.
+	AllowAutoShard bool `toml:"allow-auto-shard" json:"allow_auto_shard"`
+}
```

Comment:
Be confident with code yourself first, then the user will do!
Experimental struct seems redundant, you can comment in the doc cause the user can't see it here.